src/device.c

```diff
@@ -830,7 +830,7 @@ int device_read_data(int id)
 	struct l_queue *list;
 	int rc;
 
-	rc = modbus_read_data(thing.data_item[id].modbus_source.reg_addr,
+	rc = iface_modbus_read_data(thing.data_item[id].modbus_source.reg_addr,
 			      thing.data_item[id].modbus_source.bit_offset,
 			      &thing.data_item[id].current_val);
```

Comment:
Please allign this columns with the first parameter

Reply:
done